var/spack/repos/builtin/packages/py-radical-entk/package.py

```diff
@@ -31,6 +32,9 @@ class PyRadicalEntk(PythonPackage):
     depends_on("py-radical-utils@1.12:", type=("build", "run"), when="@1.12:")
     depends_on("py-radical-pilot@1.12:", type=("build", "run"), when="@1.12:")
 
+    depends_on("py-radical-utils@:1.11", type=("build", "run"), when="@:1.11")
+    depends_on("py-radical-pilot@:1.11", type=("build", "run"), when="@:1.11")
```

Comment:
This looks like a duplicated line?  Otherwise LGTM - thanks!

Reply:
That just to enforce particular versions (first pair for versions >=1.12, and the next pair for versions <=1.11)

Reply:
Ah, sorry, the lines are in fact different (`utils` vs. `pilot`).  Sorry for the noise...